### PR TITLE
Compliance Checker landing page

### PR DESCRIPTION
**Pull request by rps-aaron**
add compliance-checker landing page and homepage tile

### CODE REVIEW

File: views/compliance-checker.jade [hunk 1]
```diff
@@ -0,0 +1,11 @@
+extends layout
+
+block body
+  div#main-container.container.compliance-checker
+    div.row
+      div.col-md-12
+        h2 Compliance Checker 
+        p Here's your chance to check it out and provide feedback! <br/> <a href='http://data.ioos.us/compliance/index.html'>http://data.ioos.us/compliance/index.html</a>
```
Comment (Bobfrat) [inline views/compliance-checker.jade]: Can you point to https://compliance.ioos.us/index.html instead?

**Comment by Bobfrat** [comment]
Thanks @rps-aaron! For future PRs, can you fork this repo and make a PR to master against your forked feature branch?

@BeckyBaltes @mwengren @jbosch-noaa @kknee 
Here's some screenshots of the compliance checker tile and page. It's pretty boring right now. Any suggestions on content? We could use a screenshot of a checker report for the page if you'd like.

### Tile (bottom right):
screen shot 2018-06-05 at 7 00 20 pm

### Page:

screen shot 2018-06-05 at 6 59 41 pm

**Comment by jbosch-noaa** [comment]
That's a good start.  I forwarded this to our comms person to get her take on it.  The check report page is IMHO also kinda boring.  Lets see what Kate comes up with. ... I'll give her until the end of the week.

File: views/compliance-checker.jade (hunk 1 above)
Comment (rps-aaron) [reply views/compliance-checker.jade]: this is all set and pushed to the branch.

**Comment by Bobfrat** [comment]
@jbosch-noaa any updates from Kate?

**Comment by jbosch-noaa** [comment]
Hey, sorry ... Kate and I talked about it and decided it might be good to have just a big check mark (like the Catalog magnifying glass). Is that something you can create?

**Comment by Bobfrat** [comment]
@rps-aaron can you make the compliance checker tile similar to this but with a checkmark?

screen shot 2018-06-18 at 2 41 42 pm

**Comment by Bobfrat** [comment]
@rps-aaron can you post a screenshot?

**Comment by rps-aaron** [comment]
Here's the screenshot.
ioos-checker-tile

**Comment by jbosch-noaa** [comment]
It looks good. 

Makes me think we need an icon for RAs and to maybe update the Data Demo Center to be a computer monitor or something to bring it into step with the others.  Thoughts are welcome ...

**Comment by Bobfrat** [comment]
@jbosch-noaa what do you think about the page content? It's from the original issue here: https://github.com/ioos/ioos-us/issues/60

**Comment by jbosch-noaa** [comment]
@Bobfrat - The landing page is looking pretty good (I think).  I do agree with your suggestion (in that email conversation with Ajay Kirshnan) that we should put a disclaimer on the page, alerting folks to the fact that the warning messages simply mean you haven’t met our interpretation of that check.  

I hear folks asking the next question though .... "Does my file still follow the standards if I ignore these warnings?"  I'm not sure what the best phrasing of that disclaimer should be .... lets think on it and throw out suggestions.

**Comment by Bobfrat** [comment]
@jbosch-noaa here's my first stab:

### Disclaimer
The objective of the IOOS compliance checker is to check your file against our interpretation of select metadata standards for data providers to use as a guideline in generating compliant files. The compliance checker should not be considered the authoritative source on whether your file is 100% "compliant". Instead, we recommend that users use the results as a guide to work towards compliance.

**Comment by jbosch-noaa** [comment]
I like it.  I am curious what @mwengren, and @dpsnowden think.  If they are cool with it then I say go with it.

**Comment by Bobfrat** [comment]
@jbosch-noaa if approved, I can also add this text to the about page. https://compliance.ioos.us/about

**Comment by jbosch-noaa** [comment]
Hey @Bobfrat - I think it sounds good. I would want @mwengren and / or @dpsnowden to give the ultimate thumbs up though.

**Comment by Bobfrat** [comment]
@mwengren what do you think of the proposed content and disclaimer above for a compliance checker splash screen?

**Comment by mwengren** [comment]
Sorry for the non-response from before.  I can't guarantee I'll see every GitHub mention because I'm sort of buried in GitHub email, guess I need to work on that.

What about a slight variation?  

The objective of the IOOS Compliance Checker is to check your file against our interpretation of select dataset metadata standards to use as a guideline in generating compliant files. The compliance checker should not be considered the authoritative source on whether your file is 100% "compliant". Instead, we recommend that users use the results as a guide to work towards compliance.

**Comment by Bobfrat** [comment]
@rps-aaron can you rebase this PR and add the disclaimer text from above?

File: views/index.html [hunk 2]
```diff
@@ -94,6 +94,22 @@ <h1>Data Demo Center</h1>
               <h1>HF Radar</h1>
             </div>
           </a>
+          <a class="grid-item onexone compliance-checker aqua" href="/compliance-checker">
+            <div class="tile-wrap">
+              <h1>Compliance Checker</h1>
+              <span class="glyphicon glyphicon-ok" aria-hidden="true"></span>
+            </div>
+          </a>
+          <a class="grid-item twoxone osmc" href="//osmc.ioos.us">
```
[the remark this answers is not in the document]
Comment (rps-aaron) [reply views/index.html]: @Bobfrat Those tiles (osmc and oa) were added after I added the compliance checker tile, if I am not mistaken.  When I did the rebase, i had to resolve conflicts and I accepted the wrong changes causing the html to be broken and out of order, so i had to redo some work.

File: routes/index.js [hunk 3]
```diff
@@ -38,4 +39,14 @@ router.get('/comt/model_viewer', function(req, res, next) {
   res.end();
 });
 
+/* GET comt pr_inundation gifs */
+router.get('/comt/projects/pr_inundation/georges/2016/05/:filename', function(req, res, next) {
+  res.sendFile(path.join(__dirname, '../public/images/comt/', req.params.filename));
+});
```
[the remark this answers is not in the document]
Comment (Bobfrat) [reply routes/index.js]: Looks like this was removed in this commit by @gitchrisadams 
https://github.com/ioos/ioos-us/commit/6e8a9338767b383170dc9c23d2feacc5c127d019#diff-bb503f3d9047bcd4a5730559f1e16560

Can you create a new PR based off the latest from master with just the commits we need for this Compliance Checker splash page? You can cherry-pick the ones you need.

**Pull request closed without merging**